update docs (#63)
mulesoft/docs-visualizer: modules/ROOT/pages/runtime-fabric-setup.adoc

* If you are deploying an app for the first time, navigate to the deployment page and click *Deploy application*.
. In the *Properties* tab, set the following property in the app: `anypoint.platform.config.analytics.agent.header_injection.disabled=false`

> Additionally, you will need to use version 1.5.12 or above of the http connector in your mule 4 applications.

Reviewer: ```suggestion
Additionally, you must use version 1.5.12 or higher of the HTTP connector in your Mule 4 applications.
```

> == Verify Supported Mule Runtime Engine (Mule) Versions for Policies

Reviewer: ```suggestion

== Verify Supported Mule Runtime Engine (Mule) Versions for Policies
```
